pkg/networking/middleware/auth_header.go

```diff
@@ -69,10 +91,14 @@ func ShouldRequireAuthentication(apiUrl string, url *url.URL, additionalSubdomai
 
 }
 
-func AddAuthenticationHeader(authenticator auth.Authenticator, config configuration.Configuration, request *http.Request) error {
+func AddAuthenticationHeader(
+	authenticator auth.Authenticator,
+	config configuration.Configuration,
+	request *http.Request,
+) error {
 	apiUrl := config.GetString(configuration.API_URL)
 	additionalSubdomains := config.GetStringSlice(configuration.AUTHENTICATION_SUBDOMAINS)
-	isSnykApi, err := ShouldRequireAuthentication(apiUrl, request.URL, additionalSubdomains)
```

Comment:
As this is an exported function, I would not rename it, as project using the lib could be affected.